.github/workflows/release.yml

```
@@ -52,7 +52,7 @@ jobs:
       - name: Setup Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Can we make this 20 instead? It has a longer support window
https://endoflife.date/nodejs
```suggestion
          node-version: 20
```

---

Got it, 20 is the long term support version for now. Updated.